components/html/modules/content-newsletter-text-only/_main-1.html

```diff
@@ -0,0 +1,49 @@
+<section class="intro">
+	<article>
+		<h1 class="intro__title">Title of Enewsletter Here</h1>
```

Comment:
Add `id="intro_title"` to this element. Used with an `aria-labelledby` attribute on the containing section.

Reply:
So would it look like this: <h1 class="intro__title aria-labelledby"> or do I need to move the "aria-labelledby" somewhere else in the tag?

Reply:
Sorry, it didn't paste what I had..<h1 class="intro_title aria-labelledby">

Reply:
@jacobvenable I showed her how